test/streams/streams_channel_test.rb

```diff
@@ -286,31 +286,53 @@ class Turbo::StreamsChannelTest < ActionCable::Channel::TestCase
     end
   end
 
-  test "broadcasting morph now" do
-    options = { partial: "messages/message", locals: { message: "hello!" } }
-  
-    assert_broadcast_on "stream", turbo_stream_action_tag("morph", target: "message_1", template: render(options)) do
-      Turbo::StreamsChannel.broadcast_morph_to "stream", target: "message_1", **options
+  test "broadcasting actions with method morph now" do
+    options = { method: :morph, partial: "messages/message", locals: { message: "hello!" } }
```

Comment:
When does `:method` need to be nested within an `:attributes` Hash option, and when can it be passed at the top level?

Reply:
This was a miss on my part it should always be an optional key under attributes.